distribution/docs/src/main/resources/_contents/_introduction/introduction-contents.adoc

```diff
@@ -1,11 +1,13 @@
 
-Codice ${branding} is an open source, modular integration framework building on ${ddf-branding} with the addition of DoD/IC and NATO support capabilities.
+Codice ${cal-branding} is an open source, modular integration framework building on ${ddf-branding} with the addition of DoD/IC and NATO support capabilities.
 
-=== ${branding} Features
+==== ${cal-branding} Features
```

Comment:
if these docs were to be imported into a custom project, the `${branding}` can change to that project, whereas `${cal-branding}` should be set to resolve to the name of this project.